README.md

```diff
@@ -88,38 +88,11 @@ constructorio.autocomplete.getResults('dogs', {
 
 ### Recommendations
 
-The recommendations module can be used to retrieve item recommendation results. Responses will be delivered via a Promise. The `parameters` object is optional. Item id's may be of type string or array.
+The recommendations module can be used to retrieve item recommendation results for a supplied pod identifier. Responses will be delivered via a Promise. The `parameters` object is optional. Item id's may be of type string or array.
```

Comment:
I would change `item recommendations` here to just be `recommendations`.  Being less descriptive might be better for future compatibility.

Reply:
Might change `supplied pod identifier` to `given pod`

Reply:
👍